shell/platform/darwin/ios/framework/Source/accessibility_text_entry.mm

```diff
@@ -0,0 +1,404 @@
+  // requirements change.
+  return nil;
+}
+
+- (UITextWritingDirection)baseWritingDirectionForPosition:(UITextPosition*)position
+                                              inDirection:(UITextStorageDirection)direction {
+  // Not editable. Does not apply.
+  return UITextWritingDirectionNatural;
+}
+
+- (void)setBaseWritingDirection:(UITextWritingDirection)writingDirection
+                       forRange:(UITextRange*)range {
+  // Not editable. Does not apply.
+}
+
+- (CGRect)firstRectForRange:(UITextRange*)range {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return CGRectZero;
+}
+
+- (CGRect)caretRectForPosition:(UITextPosition*)position {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return CGRectZero;
+}
+
+- (UITextPosition*)closestPositionToPoint:(CGPoint)point {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return nil;
+}
+
+- (UITextPosition*)closestPositionToPoint:(CGPoint)point withinRange:(UITextRange*)range {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return nil;
+}
+
+- (NSArray*)selectionRectsForRange:(UITextRange*)range {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return @[];
+}
+
+- (UITextRange*)characterRangeAtPoint:(CGPoint)point {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+  return nil;
+}
+
+- (void)insertText:(NSString*)text {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+}
+
+- (void)deleteBackward {
+  // This method is required but not called by accessibility API for
+  // features we are using it for. It may need to be implemented if
+  // requirements change.
+}
+
+@end
+
+
+@implementation TextInputSemanticsObject {
+  FlutterInactiveTextInput* _inactive_text_input;
+}
+
+- (instancetype)initWithBridge:(shell::AccessibilityBridge*)bridge uid:(int32_t)uid {
+  self = [super initWithBridge:bridge uid:uid];
+
+  if (self) {
+    _inactive_text_input = [[FlutterInactiveTextInput alloc] init];
+  }
+
+  return self;
+}
+
+- (void)dealloc {
+  [_inactive_text_input release];
+  [super dealloc];
+}
+
+#pragma mark - UIAccessibility overrides
+
+- (void)setSemanticsNode:(const blink::SemanticsNode*)node {
```

Comment:
nit: I believe this method and the next two are not really part of UIAccessibility protocol as indicated by the pragma above?